tests/defaults_list/test_defaults_tree.py

```diff
@@ -682,6 +682,19 @@ def test_override_option_from_defaults_list(
             ),
             id="two_group_defaults_different_pkgs:override_second",
         ),
+        param(
+            "two_group_defaults_different_pkgs_global",
+            [],
+            DefaultsTreeNode(
+                node=ConfigDefault(path="two_group_defaults_different_pkgs_global"),
+                children=[
+                    GroupDefault(group="group1", value="file1"),
+                    GroupDefault(group="group1", value="file2", package="_global_"),
```

Comment:
I've noticed `package="_global_"` showing up in the `GroupDefaults` for the new tests I'm adding to this file.
In all the other tests, a global package shows up as `package=""`.
Is this inconsistency something I should try to fix?